views/update/update-manage-trusts-individuals-or-entities-involved.html

```diff
@@ -11,16 +11,209 @@
+        } %}
+        {% if trustee.ch_references %}
+          {% set reviewedFormerTrustees = (reviewedFormerTrustees.push(row), reviewedFormerTrustees) %}
+        {% else %}
+          {% set addedFormerTrustees = (addedFormerTrustees.push(row), addedFormerTrustees) %}
+        {% endif %}
+      {% endfor %}
+
+      {% set reviewedIndividualTrustees = [] %}
+      {% set addedIndividualTrustees = [] %}
+      {% for trustee in pageData.individualTrusteeData %}
+        {% set type = pageData.trusteeType.INDIVIDUAL | capitalize %}
+        {% set name = trustee.forename + " " + trustee.surname %}
+        
+        {% set row = {
+          key: { text: type },
+          value: { text: name },
+          actions: {
+            items: [CREATE_CHANGE_LINK(
+              OE_CONFIGS.UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_INDIVIDUAL_URL + "/" + trustee.id + OE_CONFIGS.FORENAME,
+              "Individual Trustee " + name,
+              "change-individual-trustee-button"
+            )]
+          }
+        } %}
+        {% if trustee.ch_references %}
+          {% set reviewedIndividualTrustees = (reviewedIndividualTrustees.push(row), reviewedIndividualTrustees) %}
+        {% else %}
+          {% set addedIndividualTrustees = (addedIndividualTrustees.push(row), addedIndividualTrustees) %}
+        {% endif %}
+      {% endfor %}
+
+      {% set reviewedLegalEntityTrustees = [] %}
+      {% set addedLegalEntityTrustees = [] %}
+      {% for trustee in pageData.trustees %}
+        {% set type = pageData.trusteeTypeTitle[trustee.trusteeItemType] %}
+        {% set row = {
+          key: { text: type },
+          value: { text: trustee.name },
+          actions: {
+            items: [CREATE_CHANGE_LINK(
+              OE_CONFIGS.UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_LEGAL_ENTITY_URL + "/" + trustee.id + OE_CONFIGS.LEGAL_ENTITY_NAME,
+              "Legal Entity Trustee " + trustee.name,
+              "change-legal-entity-trustee-button"
+            )]
+          }
+        } %}
+        {% if trustee.ch_references %}
+          {% set reviewedLegalEntityTrustees = (reviewedLegalEntityTrustees.push(row), reviewedLegalEntityTrustees) %}
+        {% else %}
+          {% set addedLegalEntityTrustees = (addedLegalEntityTrustees.push(row), addedLegalEntityTrustees) %}
+        {% endif %}
+      {% endfor %}
+
+      {% if reviewedBoItems | length or reviewedFormerTrustees | length or reviewedIndividualTrustees | length or reviewedLegalEntityTrustees | length %}
+        <h1 class="govuk-heading-m">
+          What you have reviewed for this trust
+        </h1>
+        {{ govukSummaryList({
+          rows: reviewedBoItems
+        }) }}
+
+        {{ govukSummaryList({
+          rows: reviewedFormerTrustees
+        }) }}
+
+        {{ govukSummaryList({
+          rows: reviewedIndividualTrustees
+        }) }}
+
+        {{ govukSummaryList({
+          rows: reviewedLegalEntityTrustees
+        }) }}
+
+      {% endif %}
```

Comment:
For now I'd say since this view is shared with registration and update trusts journey, that issue you've highlighted @conuirchgov is inherited/existing and after discussing a similar problem in my ticket with Matthew, the approach we should go with here is leave it as is, but Matthew will raise a separate ticket to fix it. 
As impacting registration needs discussed with their teams etc .. 

So @cdrainCH can highlight this to Matthew.

Reply:
All good :thumbsup: - was just a suggestion to try reduce code duplication knowing the other journeys have the same issue, but also because it's a separate file to registration/add so wouldn't have impacted them.

Reply:
Naw your defs right Conuir, saw on demo with Christy Friday afternoon and the big gap doesn't look good and if its a new view we need to resolve it @cdrainCH